src/Machete/Values/EntityValueList.cs

```diff
@@ -51,6 +51,9 @@ public bool TryGetValue(int index, out Value<TValue> value)
                 index >= _values.Count && _slice.TryGetSlice(i, out textSlice);
                 i++)
             {
+                if (textSlice.SourceSpan.IsEmpty)
```

Comment:
Check out ValueListTests.Should_be_able_to_. It checks whether or not the Count is correct and whether or not I can still extract each item from the list.